packages/flutter/test/rendering/object_test.dart

```diff
@@ -18,9 +18,14 @@ void main() {
     // Initialize all bindings because owner.flushSemantics() requires a window
     final TestRenderObject renderObject = TestRenderObject();
     int onNeedVisualUpdateCallCount = 0;
-    final PipelineOwner owner = PipelineOwner(onNeedVisualUpdate: () {
-      onNeedVisualUpdateCallCount +=1;
-    });
+    final PipelineOwner owner = PipelineOwner(
+      onNeedVisualUpdate: () {
+        onNeedVisualUpdateCallCount +=1;
+      },
+      onSemanticsUpdate: (ui.SemanticsUpdate update) {
```

Comment:
Not sure what the right way to dispatch this semantics update to the correct view is.

Reply:
This would be `TestRenderingFlutterBinding.instance.window` - but I don't think it is relevant for the test that that's wired up correctly. You could instead just count a onSemanticsUpdateCount up and assert that this method gets called during `flushSemantics` below.

(The PR is missing a test for this anyways)